src/assets/RawAsset.js

```diff
@@ -18,9 +18,16 @@ class RawAsset extends Asset {
     );
 
     return {
-      js: `module.exports=${JSON.stringify(pathToAsset)};`
+      js: `module.exports=${JSON.stringify(pathToAsset+'?'+this.generateHash(true))};`
     };
   }
+
+  generateHash(regenerate) {
+    if (regenerate || !this.hash) {
+      this.hash = ('t=' + Date.now());
```

Comment:
Why don't u just hash Date.now(), this is a bit confusing imo as it is not a hash but a timestamp?